### DISCUSSION TITLE
New to Tailwind, cannot get it too work...

### DISCUSSION THREAD

**Discussion by Bay102** [tailwindlabs/tailwindcss#11348] [Help]
Im using vite for a react app, ive followed the setup instructions carefully, still cant seem to get it to work. 
Any suggestions ? 

tailwind version - 3.3.2

**Comment by wongjn** [comment]
Consider checking you have followed [the installation instructions](https://tailwindcss.com/docs/guides/vite) correctly. Otherwise, consider providing a remotely-hosted git repository of the project such that we can troubleshoot any misconfiguration more concretely.

**Reply by Bay102** [reply]
Thanks Wongjn !!! i got it working :)
